packages/devtools/web/styles.css

```diff
@@ -867,19 +869,35 @@ span.strong {
     }
 }
 
+@media all and (max-width: 1000px) {
+    .btn-group.collapsible-1000 .optional-text span:not(.octicon) {
+        display: none;
+    }
+
+    .btn-group.collapsible-1000 .btn .flutter-icon {
+        margin-right: 0;
+        box-sizing: content-box;
+    }
+}
+
 @media all and (max-width: 1200px) {
-    .btn-group.collapsible-1200 .optional-text span {
+    .btn-group.collapsible-1200 .optional-text span:not(.octicon) {
         display: none;
     }
 
     .btn-group.collapsible-1200 .btn .flutter-icon {
         margin-right: 0;
         box-sizing: content-box;
     }
+
+    .btn-group.collapsible-1200 .btn .flutter-icon {
```

Comment:
Isn't this rule identical to the one right above it?

Reply:
Yeah, I wanted to be extra sure it worked 😏

(Gone now)